monitoring/managing-metrics-targets.adoc

```diff
@@ -0,0 +1,30 @@
+[id="managing-metrics-targets"]
+= Managing metrics targets
+include::modules/common-attributes.adoc[]
+:context: managing-metrics-targets
+
+toc::[]
+
+{product-title} Monitoring collects metrics from targeted cluster components by scraping data from exposed service endpoints. 
+
+In the Administrator perspective in the {product-title} web console, you can use the *Metrics Targets* page to view, search, and filter the endpoints that are currently targeted for scraping. 
+For example, you can view the current status of targeted endpoints to see when {product-title} Monitoring is not able to scrape metrics from a targeted component. 
+
+The *Metrics Targets* page shows targets for default {product-title} projects and for user-defined projects. 
+
+// Accessing the Metrics Targets UI in the Administrator perspective
+include::modules/monitoring-accessing-the-metrics-targets-page.adoc[leveloffset=+1]
+
+// Searching and filtering metrics targets
+include::modules/monitoring-searching-and-filtering-metrics-targets.adoc[leveloffset=+1]
+
+// Getting detailed information about metrics targets
+include::modules/monitoring-getting-detailed-information-about-a-target.adoc[leveloffset=+1]
+
+.Additional resources
+
+*  xref:../monitoring/managing-metrics.adoc#understanding-metrics_managing-metrics[Managing metrics]
+
+== Next steps
```

Comment:
I don't see `Next steps` anywhere on the preview link. I don't think you need an `additional resources` and a `next steps` section. Additional resources suffice sufficiently.

Reply:
I understand your point and agree in general, but in this specific instance I've added these sections is to stay consistent with similar assembly topics in the TOC

Reply:
Sounds good! Consistency is key, so your decision makes good sense!